torch/csrc/jit/codegen/cuda/scheduler/registry.cpp

```diff
@@ -355,6 +357,50 @@ class SchedulerTopologyChecker {
     return true;
   }
 };
+
+bool isConnectedFusionGraph(Fusion* fusion) {
+  if (fusion->outputs().empty()) {
+    // Trivial case interpreted as connected
+    return true;
+  }
```

Comment:
I think what you built is that empty kernels will just not do anything, is that correct? If only an empty kernel is sent to fusion executor cache will it also be skipped?

Reply:
This is boundary case handling. An empty graph is considered "connected" in this check. 

Skipping empty fusion looks like a separate issue to address. There are a few cases we need to define the right behavior. Thanks for pointing this out 👍 . Tracking this in #1385, with more details and will handle in a separate PR.